homeassistant/components/netatmo/light.py

```diff
@@ -79,7 +89,7 @@ def __init__(
         self._id = camera_id
         self._home_id = home_id
         self._model = camera_type
-        self._device_name = self._data.get_camera(camera_id).get("name")
+        self._device_name: str = self._data.get_camera(camera_id).get("name", UNKNOWN)
```

Comment:
If we don't know the device name in `device_info` we don't set it at all or set it to `None`.

Reply:
Thanks. I've addressed this in https://github.com/home-assistant/core/pull/53524.